pg_analytics/src/lib.rs

```diff
@@ -27,8 +29,16 @@ pub extern "C" fn _PG_init() {
 
     #[allow(static_mut_refs)]
     unsafe {
-        register_hook(&mut PARADE_HOOK)
-    };
+        register_hook(&mut PARADE_HOOK);
+
+        // Custom WAL manager is only supported on PG15+
+        // https://paquier.xyz/postgresql-2/postgres-15-custom-rmgr/
+        #[cfg(feature = "pg15")]
+        pg_sys::RegisterCustomRmgr(CUSTOM_RMGR_ID, &mut *CUSTOM_RMGR);
+
+        #[cfg(feature = "pg16")]
+        pg_sys::RegisterCustomRmgr(CUSTOM_RMGR_ID, &*CUSTOM_RMGR);
+    }
```

Comment:
Could you add a note about the ramifications for postgres versions <15?

Reply:
done